pywebdriver/plugins/escpos_driver.py

```diff
@@ -256,19 +271,7 @@ def get_status(self, **params):
                 # Maybe we could do something here to start serial
                 status = "connected"
             elif device_type == "usb":
-                try:
-                    if self.is_online():
```

Comment:
Is the `is_online` function still useful ?

If not maybe we should remove it

Reply:
Not really but it's a function from escpos.printer.Usb.

Reply:
Oh I assumed it was from pywebdriver sorry